firestore-palm-gen-text/extension.yaml

```diff
@@ -133,7 +133,7 @@ params:
   - param: RESPONSE_FIELD
     label: Response Field
     description: >-
-      The field in the message document into which to put the response.
+      The field in the message document into which to put the response.There's a token limit for the response field. This means responses may be truncated if they exceed a certain length. The standard limit is approximately 4500 characters, but this can vary depending on language and context. For more information, check [here](https://ai.google.dev/models/palm).
```

Comment:
```suggestion
      The field in the message document into which to put the response. There is a token limit for the response field. This means responses may be truncated if they exceed a certain length. The standard limit is approximately 4096 characters, but this can vary depending on language and context. For more information, check [here](https://ai.google.dev/models/palm).
```